### ISSUE TITLE
High resolution video takes some time to play after loading done

### ISSUE THREAD

**Issue by ganpats** [TextureGroup/Texture#1947]
While using high resolution video, video loads and after that video not start playing, it's take some time and then start playing.
I tested with below video URL on my code as well as in sample provided, but it takes 7-16 seconds before start play.
`https://storage.googleapis.com/pao-uat-6b24c.appspot.com/users/IhPOkunZoRvt4P0XIpmd/spots/Dy8Hy1w36WiiCohoX0I3/media/2GmuYtzQHJAh0zbU4OCT/2GmuYtzQHJAh0zbU4OCT?alt=media&token=1f031e81-1ee8-4da5-b148-9bee787b9e58`

Using latest version: 3.0

Thanks!

**Comment by durn3v** [comment]
@ganpats did you find a solution for this? I also have this problem, and can't find any related information about it

**Comment by ganpats** [comment]
@durn3v  no, I don't find any solution yet! I have to switch to another library.

**Comment by durn3v** [comment]
@ganpats could you tell me please, which library did you switch to?

**Comment by ganpats** [comment]
@durn3v I used [Player](https://github.com/piemonte/Player) library.
